README.md

```
@@ -104,59 +101,57 @@ The container comes with:
 - Set breakpoints as needed.
 - Click on **Run and Debug** from the main method or use `ctrl+shift+D` to jump to debug menu.
 
 > [!NOTE]
-> It is possible to debug with any IDE including Neovim, Emacs and etc but we recommend vscode for easier setup. 
+> It is possible to debug with any IDE including Neovim, Emacs and etc but we recommend Visual Studio Code for easier setup. 
 
 ## Contributing
 
-Run all formatter, lint, and test checks locally before opening a pull request:
+- Changes should be developed and push to following branches based on the area of the feature.
+    - feature/linter-core: Changes to the linter engine (lexer, parser, semantic analyzer and BNF document.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)